### ISSUE TITLE
Create Minio storage for logs, metrics, and open telemetry

### ISSUE THREAD

**Issue by computate** [nerc-project/operations#826]
## Motivation
NooBaa object storage has never been stable in our cluster, and is forcing us to choose another solution for object storage within OpenShift. 

## Completion Criteria
Deploy Minio with enough storage for Loki Logs, ACM Observability metrics, and Open Telemetry for AI Telemetry. 

## Description
- Discover scaling options for object storage with Minio
- Deploy Minio in the test cluster
- Create a Job to create a bucket for Open Telemetry in the test cluster
- Test Open Telemetry and Tempo in the test cluster and connect to the Minio bucket in the test cluster
- Deploy Minio in the infra cluster
- Create a Job to create a bucket for ACM Observability metrics in the infra cluster
- Deploy Minio in the obs cluster
- Create a Job to create a bucket for Loki Logs in the obs cluster
- Create a Job to create a bucket for Open Telemetry in the obs cluster

## Completion dates
Desired - end of 2024

**Comment by computate** [comment]
Thanks to @larsks , we will use [this as an example](https://github.com/larsks/minio-k8s/tree/feature/dex) for dex cluster authentication access control to the minio console.

**Comment by joachimweyl** [comment]
@computate my understanding is that the minIO buckets were created is this accurate?

**Comment by computate** [comment]
Yes, we have created buckets for logs, metrics, and open telemetry. We can close this issue.
